assets/js/base/context/cart-checkout/checkout/index.js

```diff
@@ -195,6 +199,8 @@ export const CheckoutProvider = ( {
 		onCheckoutProcessing,
 		dispatchActions,
 		isEditor,
+		orderId: checkoutState.orderId,
+		hasDraftOrder: !! checkoutState.orderId,
```

Comment:
So when `orderId === 0` that means this is a draft order?

Reply:
No, draft orders have ids. !! just ensures we have a bool. Could have wrote it like `checkoutState.orderId > 0`

Reply:
Hmm, the name of the property seems to suggest that there might be an orderId but it might not be for a draft order.  Could this just be `hasOrder`?